src/main/scala/mesosphere/marathon/state/AppDefinition.scala

```diff
@@ -85,7 +85,9 @@ case class AppDefinition(
 
     override val killSelection: KillSelection = KillSelection.DefaultKillSelection,
 
-    tty: Option[Boolean] = AppDefinition.DefaultTTY) extends RunSpec
+    tty: Option[Boolean] = AppDefinition.DefaultTTY,
+
+    role: Option[String] = AppDefinition.DefaultRole) extends RunSpec
```

Comment:
I don't think this should be optional.

Reply:
In that case, what do we use for DefaultRole? Or do we just not provide a default role and change every invocation to provide some role?

Reply:
Non-Optional now